src/components/password/password.pw.tsx

```diff
@@ -575,3 +577,25 @@ test("should have the expected border radius styling", async ({
   await mount(<PasswordComponent />);
   await expect(passwordInput(page)).toHaveCSS("border-radius", "4px");
 });
+
+test("should set aria-live attribute on Character Count to `polite` when component is focused and then change back to `off` when component is blurred", async ({
```

Comment:
**question**: is there a particular reason why this has been added for `Password` specifically? As it renders `Textbox` i'm happy just having this test included there

Reply:
Yeah I added it here just to catch any potential issues if we were to make any changes to `Password` in the future. `Password` was also one of the components used in the example provided in the Github issue that highlighted this bug. I felt it was best to cover all the components that used a `characterLimit` just to be cautious. I can remove the test from `Password` if we feel it's a bit OTT.

Reply:
Fair, in theory `Password` only has this functionality because it renders `Textbox`, same can be said for `Decimal`, `GroupedCharacter`, `Number` etc. I think just having a test for `Textbox` would be okay

Reply:
I'll have it removed 🙅🏻